lnrpc/walletrpc/walletkit.proto

```diff
@@ -220,46 +220,54 @@ message BumpFeeRequest {
 message BumpFeeResponse {
 }
 
+message SharedKeyRequest {
+    bytes ephemeral_pubkey = 1;
```

Comment:
Does it need to be ephemeral? I think it could be any pubkey? We wouldn't be able to verify that it is ephemeral anyway.

Reply:
This is more of a usage recommendation to signal that ephemeral keys _should_ be used.

Reply:
Doesn't that depend on the use case? The other key isn't ephemeral either, while there are applications where both keys are ephemeral.